connectors/source.py

```diff
@@ -92,6 +124,14 @@ def get(self, key, default=None):
     def has_field(self, name):
         return name in self._config
 
+    def any_field_is_empty(self, names):
+        if any(not self.has_field(name) for name in names):
+            raise ConfigurationFieldMissingError
+
+        empty_fields = [name for name in names if self._config[name].is_empty()]
+
+        return len(empty_fields) > 0, empty_fields
+
```

Comment:
This can probably be a separate PR with separate test

Reply:
It's tested in `test_any_field_is_empty` and two other tests in `test_source.py`. I can create a separate PR for the new error classes and this functionality 👍 Just don't want to overload our open PRs. We already have ~ 15 PRs open since one week.

Reply:
Yeah makes sense, just a note for the future :) I love small PRs and take much less time approving them :)